src/spyglass/common/common_nwbfile.py

```diff
@@ -667,4 +678,48 @@ def nightly_cleanup():
         # during times when no other transactions are in progress.
         AnalysisNwbfile.cleanup(True)
 
-        # also check to see whether there are directories in the spikesorting folder with this
+    def log(self, analysis_file_name, table=None):
+        """Passthrough to the AnalysisNwbfileLog table. Avoid new imports."""
+        if isinstance(analysis_file_name, dict):
+            analysis_file_name = analysis_file_name["analysis_file_name"]
+        time_delta = time() - self._creation_times[analysis_file_name]
+        file_size = os.path.getsize(self.get_abs_path(analysis_file_name))
```

Comment:
Do we know how well `getsize` measures file size? I know sometimes there are issues.

Reply:
I wasn't aware of those issues. I can look into alternatives

Reply:
I would just request a look around to see if there is anything else preferred. This is probably good enough.

Reply:
It looks like folks have a preference for pathlib over os for just being more object oriented, but not because it does the task better. I chose to go with what DJ uses in the externals table. If we have to compare for w/e reason, the numbers should match. 

If we really wanted to optimize, I think I could insert nulls and we would fetch from the externals table later, or fk ref it, but that might be a lot of hoops for a temporary feature